src/components/KTableView/KTableView.vue

```diff
@@ -612,35 +606,26 @@ const getColumnSlotName = (columnKey: string): TableColumnSlotName => {
  * This helps TypeScript infer the slot name in the template section so that the slot props can be resolved.
  * @param {string} columnKey The column.key
  */
-const getColumnTooltipSlotName = (columnKey: string): TableColumnTooltipSlotName => {
+const getColumnTooltipSlotName = (columnKey: ColumnKey): TableColumnTooltipSlotName<ColumnKey> => {
   return `tooltip-${columnKey}`
 }
 
-/**
- * To avoid requiring the consuming app to typecast if they want to use `row` or `column`
- * we strip the types to something generic before we put it in the slot for use.
- * @param obj The object to strip the type from
- */
-const getGeneric = (obj: Record<string, any>): any => {
-  return obj as unknown as any
-}
-
 /**
  * Grabs listeners from attrs matching a prefix to attach the
  * event that is dynamic. e.g. `v-on:cell:click`, `@row:focus` etc.
- * @param {String} prefix - event listener prefix e.g. `row:`, `cell:`
- * @param {any} attrs - attrs on the vue instance to pluck from
+ * @param {'onRow:' | 'onCell:'} prefix - event listener prefix e.g. `row:`, `cell:`
+ * @param {Record<string, unknown>} listenerProps - attrs on the vue instance to pluck from
  * @returns {Function} - returns a function that can pass an entity to the
                          listener callback function.
 */
-const pluckListeners = (prefix: any, attrs: any): any => {
-  return (entity: any, type: any) => {
+const pluckListeners = (prefix: 'onRow:' | 'onCell:', listenerProps: Record<string, unknown>): any => {
+  return (entity: unknown, type: 'row' | 'cell') => {
     const onRE = /^on[^a-z]/
     const listeners = {} as any
 
-    for (const property in attrs) {
-      if (onRE.test(property) && !!attrs[property]) {
-        listeners[property] = attrs[property]
+    for (const property in listenerProps) {
+      if (onRE.test(property) && !!listenerProps[property]) {
```

Comment:
nit: can you rename `onRe` to something more readable (I realize it's out of scope for PR)

Reply:
It's probably copied from Vue's old implementation: https://github.com/vuejs/core/blob/5a7a1b8293822219283d6e267496bec02234b0bc/packages/shared/src/index.ts#L40-L41 😅